docs/configuration.md

```diff
@@ -109,6 +109,12 @@ redis:
 		maxidle: 16
 		maxactive: 64
 		idletimeout: 300s
+uploadpurging:
```

Comment:
Possibly.  If the other maintenance functions (verify blobs, verify links) could also live under that umbrella, I'm for it.
```
storage:
   maintenance:
      purgeuploads:
         ...
      verification:
         ...
```